ui/test/e2e/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "e2e",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Yaraslau Kurmyza <lotask@gmail.com>",
+  "license": "ISC",
```

Comment:
Could we set this to `MPL-2.0` for consistency across the other `package.json` files?

Reply:
sure, forgot to update that file. Thanks